src/library_glut.js

```diff
@@ -361,9 +367,11 @@ var LibraryGLUT = {
     window.addEventListener("mousedown", GLUT.onMouseButtonDown, true);
     window.addEventListener("mouseup", GLUT.onMouseButtonUp, true);
     // IE9, Chrome, Safari, Opera
-    window.addEventListener("mousewheel", GLUT.onMouseWheel, true);
+    window.addEventListener("mousewheel", GLUT.onMouseWheel, { capture: true, passive: false });
     // Firefox
-    window.addEventListener("DOMMouseScroll", GLUT.onMouseWheel, true);
+    window.addEventListener("DOMMouseScroll", GLUT.onMouseWheel, { capture: true, passive: false });
+
+    window.addEventListener("resize", GLUT.onResize, true);
 
     Browser.resizeListeners.push(function(width, height) {
```

Comment:
Do we really need both a resizeListener *and* a listener on the window itself? I'm not sure why the resizeListener isn't enough by itself. (If I'm missing something here, maybe adding a comment here would help other people reading the code in the future?)

Reply:
The bug i was hitting is that the reshape function would not be called.

I have a canvas element styled with CSS to 100% width/height, so a simple resizing of the window would update the size of the canvas element.

I am not too familiar with what the library_browser.js does out of the box, but it seems like it does not handle those resizes of the canvas.

Reply:
I see, thanks.

Reading the existing code some more, it looks like none of the other input systems (sdl, etc.) listens for window resize events. And so this looks like a general issue, that maybe the resizeListeners should be called for.

And IIUC the CSS you describe, then your canvas spans the whole window, so I think a resize event should be fired on the canvas, if the window is resized. So listening on the canvas should be enough - but it looks like we don't do that either - we just look for fullscreen resizing.

In summary how about if in library_browser.js, in the `init` function, we add an event listener for the resize event, at the very end to the canvas, that calls `Browser.updateResizeListeners` (alongside the other events listened to in the block with `if (canvas) {`. Would that fix your use case? If so I think it might be a more general fix.

Reply:
That would fix my case.

I will implement the more general fix then (within this PR I guess, so we can keep the discussion).

Reply:
Sounds good! Yes, in this same PR is simplest and best, I think.